Genre/genre.rb

```diff
@@ -0,0 +1,15 @@
+require '../item'
+
+class Genre < Item
+  attr_reader :item
+
+  def initialize(name, genre)
+    super(genre)
```

Comment:
I think it would be better if you use `id:nil` in parameter reference to the constructor we are using in `Item` class and remove the `genre` reference to UML diagram. For example, something like below
```
def initialize(name, id:nil)
  @id = id.nil? ? Random.rand(1..1000) : id
  @name= name.to_s unless name.is_a? String
  @items= []
end
```

Similarly, add getter for id and set setters and getters for items and name. something like below
```
attr_reader :id
attr_accessor :name, :items
```